Integrations/Akamai_WAF/Akamai_WAF.py

```diff
@@ -0,0 +1,528 @@
+""" IMPORTS """
+# Std imports
+from datetime import datetime
+import time
+
+# 3-rd party imports
+from typing import Dict, Tuple, Union, Optional, List, Any, AnyStr
+import urllib3
+from akamai.edgegrid import EdgeGridAuth, EdgeRc
+
+# Local imports
```

Comment:
this is not according to template, why do we need this comment ?

Reply:
This is by PEP8, so it will be clear 
https://www.google.com/search?q=imports+order+python&oq=imports+order+p&aqs=chrome.0.0j69i57j0l3j69i64l3.6535j0j7&sourceid=chrome&ie=UTF-8